python/GafferUI/PathListingWidget.py

```diff
@@ -56,15 +56,15 @@
 # allows customisable column listings, and supports both single and multiple selection.
 class PathListingWidget( GafferUI.Widget ) :
 
-	Column = _GafferUI._PathListingWidgetColumn
-	StandardColumn = _GafferUI._PathListingWidgetStandardColumn
-	IconColumn = _GafferUI._PathListingWidgetIconColumn
+	Column = _GafferUI.PathColumn
+	StandardColumn = _GafferUI.StandardPathColumn
+	IconColumn = _GafferUI.IconPathColumn
```

Comment:
I hadn't really considered that to be honest. If it weren't for the need to derive from them in C++, I would probably have kept these as the primary names. But since they are available in C++, perhaps we should follow the usual practice and keep the Python API mirroring the C++ one as closely as we can? So these are deprecated?